### PR TITLE
Old Claim Commands Deprecation (Phase 2)

### LINKED ISSUE #1646: Make old claim commands only show warning message
Phase 2 of deprecation - old claim commands should show the warning message without executing the command

### PR DESCRIPTION
**Pull request by dmccoystephenson**
Made the old claim commands show the deprecation warning and not execute anything.

closes #1646

### CODE REVIEW

File: src/main/kotlin/com/dansplugins/factionsystem/command/faction/MfFactionCommand.kt [hunk 1]
```diff
@@ -226,16 +226,16 @@ class MfFactionCommand(private val plugin: MedievalFactions) : CommandExecutor,
             in devAliases -> factionDevCommand.onCommand(sender, command, label, args.drop(1).toTypedArray())
             // Backwards compatibility:
             in claimAutoAliases -> {
-                sender.sendMessage("${RED}Command deprecated, use \"/mf claim auto\" instead")
-                factionClaimAutoCommand.onCommand(sender, command, label, args.drop(1).toTypedArray())
+                sender.sendMessage("${RED}Command deprecated, use \"/mf claim auto\" instead!")
```
Comment (alyphen) [inline src/main/kotlin/com/dansplugins/factionsystem/command/faction/MfFactionCommand.kt:229]: Translations?

Comment (alyphen) [inline src/main/kotlin/com/dansplugins/factionsystem/command/faction/MfFactionCommand.kt:229]: Translations?

**Review by alyphen — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/dansplugins/factionsystem/command/faction/MfFactionCommand.kt (hunk 1 above)
Comment (dmccoystephenson) [reply src/main/kotlin/com/dansplugins/factionsystem/command/faction/MfFactionCommand.kt]: Translations have been added!

File: src/main/resources/lang/lang_de_DE.properties [hunk 2]
```diff
@@ -1066,6 +1066,9 @@ CommandFactionAddMemberUnknownNewPlayerFaction=Unbekanntes neues Mitglied
 CommandFactionAddMemberSuccess=Dieser Spieler ist jetzt Mitglied der angegebenen Fraktion.
 CommandFactionHelpAddMember=/faction addmember <player> <faction> - F�gt zwangsweise ein Mitglied zu einer Fraktion hinzu.
 CommandFactionRoleSetPermissionCannotModifyRolePermissionToModifyOwnRolePermission=Du kannst die Berechtigung deiner Rolle nicht �ndern, um die Berechtigungen deiner eigenen Rolle zu �ndern.
+DeprecationWarningAutoclaim=Warnung: Die automatische Anspruchsfunktion wird in einer zuk�nftigen Version entfernt. Verwenden Sie stattdessen /f claim auto.
```
Comment (alyphen) [inline src/main/resources/lang/lang_de_DE.properties:1069]: Maybe 'Achtung' over 'Warnung', also use impolite form (everything else uses 'du')
Pass this by our translators maybe?

**Review by alyphen — CHANGES_REQUESTED** [review]

File: src/main/resources/lang/lang_de_DE.properties (hunk 2 above)
Comment (dmccoystephenson) [reply src/main/resources/lang/lang_de_DE.properties:1069]: I had ChatGPT generate these when usually I use Google Translate. That probably explains the discrepancy.

Comment (dmccoystephenson) [reply src/main/resources/lang/lang_de_DE.properties:1069]: I've posted this in the translator chat.

Google Translate suggested the following:

```
DeprecationWarningAutoclaim=Befehl veraltet. Verwenden Sie stattdessen /fclaim auto.
DeprecationWarningFillclaim=Befehl veraltet. Verwenden Sie stattdessen /f Claim Fill.
DeprecationWarningCheckclaim=Befehl veraltet. Verwenden Sie stattdessen die Anspruchsprüfung /f.
```

Comment (alyphen) [reply src/main/resources/lang/lang_de_DE.properties:1069]: What about:
DeprecationWarningAutoclaim=Befehl veraltet. Verwendest du stattdessen /f claim auto. DeprecationWarningFillclaim=Befehl veraltet. Verwendest du stattdessen /f claim fill.
DeprecationWarningCheckclaim=Befehl veraltet. Verwendest du stattdessen /f claim check.

Comment (dmccoystephenson) [reply src/main/resources/lang/lang_de_DE.properties:1069]: Sounds good to me!

Comment (dmccoystephenson) [reply src/main/resources/lang/lang_de_DE.properties:1069]: Your suggested translations have been implemented :)

**Pull request merged**